screens/Home/ContinueCard.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC } from "react";
+import { Image, ScrollView, Text, TouchableOpacity, View } from "react-native";
+import movieCards from "../../assets/MovieCardImages";
+import CommonText from "../../shared/CommonText";
+import homeStyles from "./homeStyle";
+
+interface Props {}
+
+export const ContinueCard: FC<Props> = () => {
```

Comment:
I think we could have a better name than `ContinueCard`.  `TopPicks` sounds even better